docs/Install.md

```diff
@@ -0,0 +1,98 @@
+# Installation
+
+The following will walk through the deployment of Pelorus.
+
+### Prerequisites
+
+Before deploying the tooling, you must have the following prepared
+
+* An OpenShift 3.11 or higher Environment
+* A machine from which to run the install (usually your laptop)
+  * The OpenShift Command Line Tool (oc)
+  * [Helm3](https://github.com/helm/helm/releases)
+  * jq
+
+### Deployment Instructions
+To deploy Pelorus, run the following script from within the root repository directory
+
+
+```
+./runhelm.sh
+```
+By default, Pelorus will be installed in a namespace called `pelorus`. You can customize this by passing `-n <my-namespace>` like so:
+
+```
+./runhelm.sh -n <my-namespace>
+```
+
+Pelorus also has additional (optional) exporters that can be deployed to gather additional data and integrate with external systems. Consult the docs for each exporter below:
+
+* [Commit Time Exporter](exporters/committime/README.md)
+* [Deploy Time](exporters/deploytime/README.md)

```

Comment:
hey @kenwilli these links are broken now because the content moved. I think we can instead just link to the Configuring Metrics Exporters section of the config guide, and call it good.